.github/workflows/quick.yaml

```diff
@@ -116,8 +116,8 @@ jobs:
     name: build-tests(${{ matrix.os }},${{ matrix.compiler.CC }},${{ matrix.layer.nick }})
 
     env:
-      CC: ${{ matrix.compiler.CC }}
-      CXX: ${{ matrix.compiler.CXX }}
+      CC: /usr/bin/ccache ${{ matrix.compiler.CC }}
+      CXX: /usr/bin/ccache ${{ matrix.compiler.CXX }}
```

Comment:
When things go wrong (e.g., bad/stale info gets cached as it has happened in the past on Jenkins IIRC), how to erase ccache (without changing our workflow files)? I could not quickly find instructions in Ccache Action docs, but there is probably a way to do that from the command line and/or GitHub interface?

Reply:
`gh -R squid-cache/squid cache list`
and
`gh -R squid-cache/squid cache delete <cache-key|"all">`

can help